
<launch>
<group ns = "robot_10">
  <node pkg="robots_deployment_online" type="deployment.py" name="ros_deployment_10" output="screen">
    <param name="pose_send_time" value="0.3" />
    <param name="id" value="10" />
    <param name="tree_file" value="/home/elerson/catkin_ws/src/networked_robots_experimetal_data/1euclideanexperiment/steinerData1.dat" />
    <param name="map_resolution" value="0.05" />
    <param name="ray" value="80.0"/>
  </node> 


 <node pkg="fake_localization" type="fake_localization" name="amclrobot_10" output="screen">
   <!-- Publish scans from best pose at a max of 10 Hz -->
   <param name="odom_model_type" value="omni"/>
   <param name="base_frame_id" value="robot_10/base_link"/>
   <param name="odom_frame_id" value="robot_10/odom"/>
   <param name="global_frame_id" value="/map"/>
  </node>


  <node pkg="move_base" type="move_base" respawn="false" name="move_base_10" output="screen">
   <!--<param name="base_global_planner" value="network_planner/NetworkDeploymentPlanner"/> -->

   <rosparam file="$(find robots_deployment_online)/launch/costmap_common_params.yaml" command="load" ns="global_costmap" />
   <rosparam file="$(find robots_deployment_online)/launch/costmap_common_params.yaml" command="load" ns="local_costmap" />
   <rosparam file="$(find robots_deployment_online)/launch/local_costmap_params.yaml" command="load" />
   <rosparam file="$(find robots_deployment_online)/launch/global_costmap_params.yaml" command="load" />
   <rosparam file="$(find robots_deployment_online)/launch/base_local_planner_params.yaml" command="load" />
   
   <param name="planner_file_dir" value="$(find network_deployment_planner)/experiment/exp1/"/>
   <param name="global_costmap/robot_base_frame" value="robot_10/base_footprint"/>
   <param name="local_costmap/global_frame" value="robot_10/odom"/>
   <param name="local_costmap/robot_base_frame" value="robot_10/base_footprint"/>
   <remap from="/robot_10/map" to="/map" />
  </node>
</group>

 </launch>
